python/servo/package_commands.py

```diff
@@ -96,8 +96,11 @@ def copy_dependencies(binary_path, lib_path):
 
 def copy_windows_dependencies(binary_path, destination):
     deps = [
+        "api-ms-win-crt-runtime-l1-1-0.dll",
         "libcryptoMD.dll",
         "libsslMD.dll",
+        "msvcp140.dll",
+        "vcruntime140.dll",
```

Comment:
If this is better, i don't know, but since is only additional 726 KB, is easier to include those 3 files.

`api-ms-win-crt-runtime-l1-1-0.dll` is part of Windows SDK.